auditexport/utils.py

```diff
@@ -106,10 +106,14 @@ def pre_export(fileFormat, records):
     return preparedRecords
 
 
-def exportToJson(pathToFolder, records):
+def exportToJson(pathToFolder, records, safeFilename):
     preparedRecords = pre_export('json', records)
+    date = str(datetime.datetime.utcnow().isoformat())
+    
+    if safeFilename:
+        date = re.sub(r'[^a-zA-Z0-9]+', '-', date)
```

Comment:
Can we make this default for Windows so if condition would be either of the following
- `if safeFilename or os.name == 'nt'`
- `if safeFilename or platform.system() == 'Windows'` (would have to import platform for this one)

And can we be more obvi that the hyphenated numbers are time by ditching hyphens in favor of letters (h, m, s)?
Like [we do here](https://github.com/virtru/console/blob/32c3c26d8573b96ad692b1243bb3f415fdf32566/src/scenes/AdvancedQuery/AdvancedQuery.js#L29-L32)?

Reply:
@suchak1 I took your suggestion on the `platform.system()`, but going to defer on the adding letters.

The python script will likely be phased out in favor of API connection or stream connection to the next Audit API. If we end up needing a script of some kind I'd rather rewrite it in Go and build binaries for Windows, *nix, and Macos.